external-crates/move/crates/move-package/src/compilation/compiled_package.rs

```diff
@@ -28,7 +28,7 @@ use move_compiler::{
     compiled_unit::{AnnotatedCompiledUnit, CompiledUnit, NamedCompiledModule},
     diagnostics::FilesSourceText,
     editions::Flavor,
-    shared::{NamedAddressMap, NumericalAddress, PackageConfig, PackagePaths},
+    shared::{NamedAddressMap, NumericalAddress, PackageConfig, PackagePaths, SourceFileReader},
```

Comment:
I'm also wondering if we should add this to the BuildConfig or anything? 
Like sure the source files can be virtual, but the toml file isn't. 

I'm 
A) not sure we want/need this
B) not sure how difficult it would be

mostly flagging as a concern since the compiler would now have one view of IO and the package system another

Reply:
Any thoughts @tzakian